bundles/media/imagine-adapter.rst

```diff
@@ -61,16 +61,11 @@ To install it use following commands:
     pecl install vips
     composer require rokka/imagine-vips
 
-contao/imagine-svg
-------------------
+SVG
+---
 
-This adapter only handles SVG images and is an addition to the other adapters.
-
-To install it use following command:
-
-.. code-block:: bash
-
-    composer require contao/imagine-svg
+The `contao/imagine-svg`_ adapter only handles SVG images and is an addition to the other adapters.
+It is shipped with sulu so it don't need to be installed manually.
```

Comment:
`it doesn't`

Reply:
And should we mention it at all, if it is not necessary to do anything? :thinking:

Reply:
`SVG` is not really matching with the other headings on this level.

Reply:
I can also remove it.

Reply:
I removed it from the docs.